ts/components/conversation/message/message-item/InteractionNotification.tsx

```diff
@@ -38,7 +38,7 @@ export const InteractionNotification = (props: PropsForInteractionNotification)
 
   switch (interactionType) {
     case ConversationInteractionType.Hide:
-      text = window.i18n('hideConversationFailedPleaseTryAgain');
+      // this can't happen
```

Comment:
Please add a NOTE infront for easy searching later

Reply:
I actually like it better as part of the case itself